webapp/test/initMockServer.js

```diff
@@ -1,16 +1,20 @@
 sap.ui.define([
-	"../localService/mockserver",
-	"sap/m/MessageBox"
+	"../localService/mockserver"
 ], function (mockserver, MessageBox) {
 	"use strict";
 
 	var aMockservers = [];
 
 	// initialize the mock server
 	aMockservers.push(mockserver.init());
+	
+	// load configure sap.m with async load
```

Comment:
How does the comment (`sap.m`) fit to the command (loading `sap.ui.core`)?

Reply:
@codeworrior Thx. Typo.